src/carousel.ts

```diff
@@ -22,6 +26,34 @@ function carousel(id: string, curChild=0) {
         centerCard(curChild);
     });
 
+
+    //Enable Swipe on mobile devices only
+    hammer.set({enable: windowSize <= 768 ? true : false})
+    //Detect swipe left or right on divParent
+    hammer.on("swipeleft swiperight", function(e: any) {
```

Comment:
Prefer arrow function:

(e) => {

any is not a great type declaration.  Hammer should be typed in such a way as to
not need an explicit declaration here - it should be inferred and have Hammer's
properties added to the event?